fixtures/chauvet-dj/120st.json

```diff
@@ -0,0 +1,168 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "LED Follow Spot 120ST",
+  "shortName": "120ST",
+  "categories": ["Dimmer", "Other"],
+  "meta": {
+    "authors": ["Ryan Goodwin"],
+    "createDate": "2023-05-24",
+    "lastModifyDate": "2023-05-24"
+  },
+  "links": {
+    "manual": [
+      "https://www.chauvetdj.com/wp-content/uploads/2016/09/LED_Followspot_120ST_UM_Rev3.pdf",
+      "https://www.chauvetdj.com/wp-content/uploads/2016/09/LED_Followspot_120ST_QRG_Rev2_ML6_Print.pdf"
+    ],
+    "productPage": [
+      "https://www.chauvetdj.com/products/led-followspot-120st/"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=mxAcqb5Cw1A"
+    ]
+  },
+  "physical": {
+    "dimensions": [279, 341, 582],
+    "weight": 6,
+    "power": 185,
+    "DMXconnector": "3-pin",
+    "bulb": {
+      "type": "120W LED"
+    },
+    "lens": {
+      "degreesMinMax": [5, 20]
```

Comment:
Webpage says 14 to 20 ... but it looks like the iris can stop it down to 5.

Do we care more about the size of the area, or about computing the absolute intensity in that area?  OFL doesn't really distinguish these cases.  Anyway, this file doesn't include any illuminance data, so I guess 5 makes as much sense as anything.

Reply:
I generally mark what is the absolute min and max for fixtures. That way the parameters min and max are always within the specs OFL lists.